test-suite/generated-src/java/com/dropbox/djinni/test/ConstantsInterface.java

```diff
@@ -8,62 +8,62 @@
 import javax.annotation.Nonnull;
 
 /** Interface containing constants */
-public abstract class ConstantsInterface {
-    public static final boolean BOOL_CONSTANT = true;
+public interface ConstantsInterface {
+    boolean BOOL_CONSTANT = true;
 
-    public static final byte I8_CONSTANT = 1;
+    byte I8_CONSTANT = 1;
 
-    public static final short I16_CONSTANT = 2;
+    short I16_CONSTANT = 2;
 
     /** i32_constant has documentation. */
-    public static final int I32_CONSTANT = 3;
+    int I32_CONSTANT = 3;
```

Comment:
shouldn't this continue to be public static final?

Reply:
Fields in Java interfaces are implicitly public static final.